support_files/launch/notepadqq

```diff
@@ -1,31 +1,23 @@
 #!/bin/sh
 SCRIPTPATH="$(dirname "$(readlink -f "$0")")"
 
-if [ `getconf LONG_BIT` = "64" ]
-then
+if [ "$(getconf LONG_BIT)" = "64" ]; then
     GCC_DIR=gcc_64
 else
     GCC_DIR=gcc
 fi
 
-OPT_QT510=/opt/Qt/5.10/$GCC_DIR/lib
-OPT_QT59=/opt/Qt/5.9/$GCC_DIR/lib
-OPT_QT58=/opt/Qt/5.8/$GCC_DIR/lib
-OPT_QT57=/opt/Qt/5.7/$GCC_DIR/lib
-OPT_QT56=/opt/Qt/5.6/$GCC_DIR/lib
-OPT_QT55=/opt/Qt/5.5/$GCC_DIR/lib
-OPT_QT54=/opt/Qt/5.4/$GCC_DIR/lib
-OPT_QT53=/opt/Qt/5.3/$GCC_DIR/lib
-PERSONAL_QT510=~/Qt/5.10/$GCC_DIR/lib
-PERSONAL_QT59=~/Qt/5.9/$GCC_DIR/lib
-PERSONAL_QT58=~/Qt/5.8/$GCC_DIR/lib
-PERSONAL_QT57=~/Qt/5.7/$GCC_DIR/lib
-PERSONAL_QT56=~/Qt/5.6/$GCC_DIR/lib
-PERSONAL_QT55=~/Qt/5.5/$GCC_DIR/lib
-PERSONAL_QT54=~/Qt/5.4/$GCC_DIR/lib
-PERSONAL_QT53=~/Qt/5.3/$GCC_DIR/lib
-
-export LD_LIBRARY_PATH="$OPT_QT510:$PERSONAL_QT510:$OPT_QT59:$PERSONAL_QT59:$OPT_QT58:$PERSONAL_QT58:$OPT_QT57:$PERSONAL_QT57:$OPT_QT56:$PERSONAL_QT56:$OPT_QT55:$PERSONAL_QT55:$OPT_QT54:$PERSONAL_QT54:$OPT_QT53:$PERSONAL_QT53:${LD_LIBRARY_PATH}"
+for VER in 10.1 10 9 8 7 6 5 4 3; do
```

Comment:
I'm pretty sure both Linux and MacOS support the {#..#} syntax.  You could change this to:
```sh
VER_MAJOR=5
for VER_MINOR in {11..3}; do
    opt_path="/opt/Qt/${VER_MAJOR}.${VER_MINOR}/$GCC_DIR/lib"
```

Reply:
I seems there are other dot versions too, i.e. 5.9.2. Maybe the script should use `ls` or something instead?

Reply:
I don't think the old script supported patch versions.  If you wanted to add that in I'm sure it would be a welcome addition.  No complaints from me there.